shared/src/test/mockJudgesChambers.ts

```diff
@@ -0,0 +1,237 @@
+import { JudgeChambersInfo } from '@shared/proxies/users/getJudgesChambersProxy';
+
+const JUDGES_CHAMBERS: Record<string, JudgeChambersInfo> = {
```

Comment:
To be pared down or removed after we run `scripts/run-once-scripts/add-phone-numbers-to-judge-records.ts`